website/sidebars.js

```diff
@@ -87,6 +87,7 @@ module.exports = {
             'plugins/colorlog',
             'plugins/joblib_launcher',
             'plugins/nevergrad_sweeper',
+            'plugins/ray_launcher',
```

Comment:
At some point we will want to group plugins by type in the sidebar.

Reply:
sounds good!